recipe/meta.yaml

```diff
@@ -10,13 +10,18 @@ source:
   sha256: dcfbc9c528bcd9e4f9686e673956a90172826395ac5b258039e580777d50782f
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
+    # Old style CLI
     - dask-scheduler = distributed.cli.dask_scheduler:main
     - dask-ssh = distributed.cli.dask_ssh:main
     - dask-worker = distributed.cli.dask_worker:main
+    # New CLI added in https://github.com/dask/distributed/pull/6735
+    - dask scheduler = distributed.cli.dask_scheduler:main
+    - dask ssh = distributed.cli.dask_ssh:main
+    - dask worker = distributed.cli.dask_worker:main
```

Comment:
Idk if doing this with a space works as expected. What happens here is a binary is created that has a space in its name. For example...

```sh
$ which "dask scheduler"
/Users/jkirkham/miniforge/envs/dask/bin/dask scheduler
```

Though this isn't the same as...

```sh
$ which dask
/Users/jkirkham/miniforge/envs/dask/bin/dask
```

...or...

```sh
$ which dask scheduler
/Users/jkirkham/miniforge/envs/dask/bin/dask
scheduler not found
```

Should add `dask scheduler` with a space doesn't seem to work correctly either. So there is still some work needed here (Idk what atm).

```sh
dask scheduler --help
Usage: dask [OPTIONS] COMMAND [ARGS]...
Try 'dask -h' for help.

Error: No such command 'scheduler'.
```

Reply:
The relevant entrypoint in `distributed` is [defined here](https://github.com/dask/distributed/blob/6afce9cce6ab402fa44736dc57b39fbc355b4c11/setup.py#L70-L73). Note that they're not `console_scripts` entrypoints, but are instead using the new `dask_cli` entrypoint. In [this section](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) of the `conda-forge` docs, it says: 

> Only console_scripts entry points have to be listed in meta.yaml. Other entry points do not conflict with noarch and therefore do not require extra treatment. 

Based on that, I agree we don't need to include entries for these entrypoints in the recipe here. We included the changes in this PR and over in the `dask` feedstock at the same time. Based on these docs, maybe only the `dask` feedstock changes were needed.

Reply:
Trying that out in https://github.com/conda-forge/distributed-feedstock/pull/226

Reply:
Thinking about this more, maybe we should try dropping [the entrypoint from `dask`]( https://github.com/conda-forge/dask-feedstock/blob/c7a89482d2061652afb380f14d4df25037e0c9b1/recipe/meta.yaml#L15-L16 ) as well. Perhaps that fixes the issue?